pkg/openservicebroker/server/route.go

```diff
@@ -79,6 +83,53 @@ func contentType(req *restful.Request, resp *restful.Response, chain *restful.Fi
 	chain.ProcessFilter(req, resp)
 }
 
+/*
+	The following properties MUST appear within the JSON encoded `value`:
+
+	| Property | Type | Description |
+	| --- | --- | --- |
+	| username | string | The `username` property from the Kubenernetes `user.info` object. |
+	| uid | string | The `uid` property from the Kubenernetes `user.info` object. |
+	| groups | string | The `groups` property from the Kubenernetes `user.info` object. |
+
+	Platforms MAY include additional properties.
+
+	For example, a `value` of:
+	```
+	{
+	  "username": "duke",
+	  "uid": "c2dde242-5ce4-11e7-988c-000c2946f14f",
+	  "groups": { "admin", "dev" }
+	}
+	```
+	would appear in the HTTP Header as:
+	```
+	X-Broker-API-Originating-Identity: kubernetes eyANCiAgInVzZXJuYW1lIjogImR1a2UiLA0KICAidWlkIjogImMyZGRlMjQyLTVjZTQtMTFlNy05ODhjLTAwMGMyOTQ2ZjE0ZiIsDQogICJncm91cHMiOiB7ICJhZG1pbiIsICJkZXYiIH0NCn0=
+	```
+*/
+func getUser(req *restful.Request) (user.Info, error) {
+	identity := req.Request.Header.Get(api.XBrokerAPIOriginatingIdentity)
+	//TODO - when https://github.com/kubernetes-incubator/service-catalog/pull/939 sufficiently progresses
+	// we will error if the originating identity header is not set, so remove the identity == "" check
+	if identity == "" {
+		return &user.DefaultInfo{}, nil
+	}
+	parts := strings.SplitN(identity, " ", 2)
+	if parts[0] != api.OriginatingIdentitySchemeKubernetes || len(parts) != 2 {
```

Comment:
please double check if you should be checking against kubernetes or Kubernetes, or do a strings.ToLower